direct/io-s3/src/main/java/cz/o2/proxima/direct/s3/S3Client.java

```diff
@@ -53,6 +53,8 @@
 @Slf4j
 class S3Client implements Serializable {
 
+  private static final long serialVersionUID = 1L;
```

Comment:
This doesn't belong to this PR.

Reply:
It is not directly related to the commit, yes. I can split it into two commits, but that makes only a small difference. It is needed, because the write should have had the `serialVersionUID` defined. I'd keep it as it is.